### ISSUE TITLE
hi

### ISSUE THREAD

**Issue by drforbin** [agn453/RSTS-E#4]
Where can I get the install tapes?

**Comment by agn453** [comment]
Look on bitsavers.org at http://www.bitsavers.org/bits/DEC/pdp11/rsts/

If you're looking for a RSTS/E V10.1 install tape, you'll find it in RSTS_00B.ZIP.

Tony

**Comment by drforbin** [comment]
thxs...what are the other files related to?
Is there a key for translaton between RSTS_00B.zip and knowing it's 10.1?
Also how do I get mail  working?

**Comment by agn453** [comment]
No, there isn't a file content listing for the ZIP files.  I mentioned the RSTS_00B.ZIP file since a few people have asked for the V10.1 installation kit (which is no longer available via anonymous ftp from ftp.trailing-edge.com in the /pub/rsts_dists directory).

In the bitsavers directory there are various versions of layered product installation kits (like DECnet/E, BASIC-PLUS-2, FORTRAN 4 PLUS, FORTRAN 77, C, COBOL etc) that can mainly be installed using the ```@[0,1]instal layered_products``` command file.  Tracking down PDF versions of the product installation guides and release notes for each kit is a challenge though. I found some at http://mim.stupi.net/manuals/layered/ that may help you.

For mail under RSTS/E you'll need to install the DEC MAIL-11 V3.1 kit (using ```@[0,1]instal layered mail```) from the tape image in DECMAIL.ZIP .  I think you need DECnet/E installed and running first to send messages to other DECnet connected machines (OpenVMS, RSX-11M etc).  There's also a text file called ```NAMES.DAT``` in the directory pointed at by the ```MAIL$``` logical name where you can map RSTS project/programmer numbers to corresponding mailbox names then ```run mail$:names``` to compile this into a system database.

Tony

**Comment by drforbin** [comment]
Thxs for all your help tony. I think I have it working!
Just out of curiosity. Is the RSTS_00b.zip the same as the v10.1 install kit? and why have they removed it? Also if yoy want the install kit I found it.

**Comment by agn453** [comment]
Yes, the contents of RSTS_00B.ZIP is a single tape image file for the last official Digital release of RSTS/E V10.1 before all the PDP11 operating system rights were sold to Mentec.  There is a Y2K update that was released by Mentec that I’ve been unsuccessful in tracking down via various contacts.  The master tapes I believe were lost in a fire.  There’s a cover letter on my RSTS-E GitHub for this - which is all I have been able to find.  My original 9-track tapes were all destroyed/discarded when my employer decided they were no longer needed and we had no equipment left that could read them.

**Comment by drforbin** [comment]
I have the one from trailing edge if you want it?

**Comment by agn453** [comment]
I have that one too. Thanks.  If you ever find a copy of the Mentec updates by all means get in touch!

**Comment by drforbin** [comment]
What's the file name of mentec update?
Also, what's the diff between trailing edge and rsts00b.zip?

**Comment by drforbin** [comment]
If there a vi editor avable for pdp11?

**Comment by agn453** [comment]
According to the cover letter, the Mentec update is a tape containing a backup saveset that is restored over a normal installation of RSTS/E V10.1.  To get it running you do a patch update and rebuild/install a new system monitor.  The filename is ```RSTS.BCK``` as per the [Cover Letter](https://github.com/agn453/RSTS-E/blob/master/docs/AV-RK36A-TE-RSTS_E-V10.1A_Cover_Letter_Year_2000_Update_1999.pdf).

I have tried tracking this down by contacting Stuart Brook who worked on the updates for Mentec.  He has searched his files and boxes of tapes without finding it.  Mentec no longer exists - and rights were transferred to XX2247 and subsequently the master tapes were lost in a fire.  The only hope is finding someone that was sent a copy and they are willing to make an image of their tape.

The bitsavers tapes were obtained by Al Kossow from another source (and include source-code), whereas the trailing edge ones were made from another collection (and some are in tpc image format and not SIMH style .tap images).  I see there is a file size difference between the trailing edge and bitsavers install tape images and I'm not sure what the differences are.  My version of the latest OpenSIMH pdp11 reports record length issues when it tries to determine the tape file formats.  (The tape is unusual in that it is bootable to the install monitor and it has an "ANSI-style" format appended to it that's used to restore the system files during an install without rewinding the tape).  It also must be mounted read-only for the installation to succeed.

```
-rwxrwxrwx 1 tony tony 24096588 Feb 20  2010 rstse_v10_1_install_sep10_1992.tap
-rwxrwxrwx 1 tony tony 24057368 Nov 24  1993 rsts_v10_1_install.tap
```

I'm not aware of a version of vi for RSTS/E.  The system supplies the Digital standard editor EDT and TECO.  Both editors may be used in line mode or screen mode on a VT100-style terminal.

**Comment by drforbin** [comment]
Thank you so much.
I'm UNIX tops10/20 guy.
What do you suggest for editing edt in line mode?

**Comment by agn453** [comment]
Any serial ASCII terminal can be used.  I won't detail how to use EDT (since there's a manual that describes this).
EDT commands are entered to an asterisk prompt and followed by Return.  To exit insert mode use Ctrl-Z.

```
$ edit/nocommand file.txt
Input file does not exist
[EOB]
*i
            This is line 1
            ^Z
[EOB]
*ex
FILE  .TXT 1 line

$ type file.txt
This is line 1
$ 
```

**Comment by drforbin** [comment]
Tony, where can I find a version of decnet to install on rsts10.1?
And how do I get it working.
Thank you

**Comment by drforbin** [comment]
Any idea?

**Comment by agn453** [comment]
(It's just after 8am Sunday morning here in Australia....   just saying this in case you didn't realise that I am not online all the time).

You can get the DECnet/E V4.1 installation kit from the location I gave earlier.

Some time back I included a prebuilt image of RSTS/E here with DECnet and some language layered products pre-installed.
You can access it via a previous commit [here](https://github.com/agn453/RSTS-E/tree/9ca6985f7cadd0a403b80056e1ca2cec9aa35fe8/prebuilt).

**Comment by agn453** [comment]
> ... I see there is a file size difference between the trailing edge and bitsavers install tape images and I'm not sure what the differences are.
> 
> ```
> -rwxrwxrwx 1 tony tony 24096588 Feb 20  2010 rstse_v10_1_install_sep10_1992.tap
> -rwxrwxrwx 1 tony tony 24057368 Nov 24  1993 rsts_v10_1_install.tap
> ```

Yesterday I checked to see if I could boot the version from the RSTS_00B.ZIP file.  It appears the rsts_v10_1.install.tap image is corrupted.

**Comment by drforbin** [comment]
I'm sorry tony I was not aware of that.
You have been a great help thank you.

Yes. Some of the images on bitsavers are corrupt. They should be made aware of this.
I just retrieved decnet v4 tap I'm going to try and install it.
After I do I'd appreciate a few pointers for getting decnet up..
I really donot know much about decnet...mostly tcp/ip guy/unix here.
I did do alot of work on dec-10's including rewritting decwar

**Comment by drforbin** [comment]
Tony,

I'm trying to install decnet. It's not finding the ethernet device.
I have xq define in my boot ini file but the monitor I have running was built before I defined it.
Do I have to rebuild monitor?

**Comment by agn453** [comment]
Yes you will need to rebuild and install a new monitor to use the Ethernet device.  The system manager and installation guide describes how to do this.

Also be aware that DECnet/E under RSTS/E acts a phase IV end-node only.  If you want to communicate with other systems you’ll need them configured for the correct DECnet area and have appropriately assigned node numbers.  I’ve not found the corresponding DECnet/E V4.1 installation guide on the Internet.  To configure DECnet you might get an idea of the steps involved by referring to OpenVMS or RSX-11 DECnet manuals.

**Comment by drforbin** [comment]
Ok.all done. The monitor sees it (i.e. device XE0) and decnet installed.
I dd find the decnet install manual and manager guide. want them?

**Comment by agn453** [comment]
Yes please.  Either attach the PDF of the manual in a reply, or the link where you found it.

**Comment by drforbin** [comment]
[AA-PAFHC-TK_Pathworks_for_DOS_DECnet_Network_Management_Guide_V4.1_Sep91.pdf](https://github.com/user-attachments/files/15944928/AA-PAFHC-TK_Pathworks_for_DOS_DECnet_Network_Management_Guide_V4.1_Sep91.pdf)

[AA-K714A-TC.pdf](https://github.com/user-attachments/files/15944929/AA-K714A-TC.pdf)

**Comment by agn453** [comment]
That's not a manual for DECnet/E V4.1 under RSTS/E - instead it is for setting up DECnet under MS-DOS (part of the Pathworks for DOS software package).

**Comment by drforbin** [comment]
I didnt look at it yet

**Comment by drforbin** [comment]
Was the other of any help?

**Comment by drforbin** [comment]
How about this
[AA-K714A-TC.pdf](https://github.com/user-attachments/files/15958855/AA-K714A-TC.pdf)

**Comment by drforbin** [comment]
[AA-AV51A-TK_intrPhIII_May82.pdf](https://github.com/user-attachments/files/15958861/AA-AV51A-TK_intrPhIII_May82.pdf)

**Comment by agn453** [comment]
Neither of these are the DECnet/E V4.1 installation guides.  One is for DECnet/E V2.0 on RSTS/E V7.1 and the other is about DECnet phase III from 1982.  DECnet/E V4.1 was released for RSTS/E V9.7 in July 1989.  I have looked in many places online (including bitsavers and archive.org) without success.

**Comment by drforbin** [comment]
Im trying
